libs/core/src/components/pds-text/pds-text.tsx

```diff
@@ -0,0 +1,97 @@
+import { Component, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'pds-text',
+  styleUrl: 'pds-text.scss',
+  shadow: true,
+})
+export class PdsText {
+
+  /**
+   * Sets the text alignment.
+   */
+  @Prop() align?: 'start' | 'center' | 'end' | 'justify';
+
+  /**
+   * Sets the text color.
+   */
+  @Prop() color?:
+  | 'primary'
+  | 'secondary'
+  | 'neutral'
+  | 'accent'
+  | 'danger'
+  | 'info'
+  | 'success'
+  | 'warning';
+
+  /**
+   * Set the bottom margin for the text.
+   */
+  @Prop() gutter?:
+  | '2xl'
+  | 'xl'
+  | 'lg'
+  | 'md'
+  | 'sm'
+  | 'xs'
+  | '2xs';
+
+  /**
+   * Sets the font size.
+   */
+  @Prop() size?:
+  | '2xl'
+  | 'xl'
+  | 'lg'
+  | 'md'
+  | 'sm'
+  | 'xs'
+  | '2xs';
+
+  /**
+   * Sets the font weight.
+   */
+  @Prop() weight?:
+  | 'extra-light'
+  | 'light'
+  | 'regular'
+  | 'medium'
+  | 'semibold'
+  | 'bold';
+
+  /**
+   * Determines what semantic text tag to render.
+   */
+  @Prop() tag:
+  | 'h1'
+  | 'h2'
+  | 'h3'
+  | 'h4'
+  | 'h5'
+  | 'h6'
+  | 'p'
+  | 'code'
+  | 'pre'
+  | 'strong'
+  | 'em' = "p";
```

Comment:
I think we need to add a span for something like this

```code
 <pds-text>
    Text Message transcipt
    <pds-text>
      <pds-text tag="em">John:</pds-text> <pds-text tag="span">Hi Sally</pds-text>
      <p><pds-text tag="em">Sally:</pds-text> <pds-text tag="span">Hey John</pds-text></p>
    </pds-text>
  </pds-text>
```

in `pds-text.scss` we need to change `:host { display: block; }` to `display: inline`

![image](https://github.com/user-attachments/assets/89fc1e32-35c5-427a-8c42-0ea6493484e8)

Reply:
@ju-Skinner could we not just use those actual tags?

```<pds-text tag="p"><em>John:</em><span>Hi Sally<span></pds-text>```

Reply:
If we're supporting other inline elements, `span` does make sense as well.

Reply:
Added italics as an optional attribute as this would allow emphasized text, `em`, to be used without forced italics. This would give us another inline element with semantic meaning attached without allowing span.